package.json

```diff
@@ -112,7 +112,7 @@
 		"terser-webpack-plugin": "^5.3.6",
 		"ts-node": "^10.9.1",
 		"type-fest": "^3.5.7",
-		"typed-query-selector": "^2.8.0",
+		"typed-query-selector": "github:g-plane/typed-query-selector#6338b01",
```

Comment:
CI only fails because this hasn't been published to npm yet

Reply:
You can upgrade to v2.9.1 now.